packages/storybook/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "storybook",
+  "version": "0.0.0",
+  "description": "Storybook build for core package",
+  "private": true,
```

Comment:
Should these still be private?

Reply:
Yup, we'll build and deploy the storybook within this repo. Without private we can't use "nohoist"